### PR TITLE
add faq and about pages

### PR DESCRIPTION
**Pull request by zoff99**
This change is [Reviewable](https://reviewable.io/reviews/toktok/website/137)

### CODE REVIEW

**Comment by zoff99** [comment]
https://github.com/Tox/tox.chat/pull/208

**Comment by zoff99** [comment]
email addresses check fail on travis, but i want to keep the content synced with tox.chat

**Comment by zugz** [comment]
What's the reasoning behind this PR? Why do we want to move these pages to toktok, and is tox.chat happy with the change?

**Comment by zoff99** [comment]
@zugz we discussed this a while ago with @iphydf 
the about page needs some tweaks, not all info is still correct.
i am just copying both pages as-is.
we can talk about linking later.

**Comment by zugz** [comment]
The new versions of the pages look fine to me, in that nothing important 
has changed from the originals, except for the missing question marks in 
the questions, and a missing colon. I think they should be put back 
(even if this means rule changes).

**Comment by zoff99** [comment]
@iphydf do you want to relax the rules for headers or want me to leave it like this ("?" and ":" removed from ending of headers) ?

File: toktok/about.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+email to our [distribution list](mailto:leadership@tox.chat). Please note that this
+list is not the right place to ask for general user support. All such
+emails would simply be ignored. Please use
+our [mailing lists](https://lists.tox.chat/listinfo)
+or the [Reddit forums](https://www.reddit.com/r/projecttox/) for general user support.
+
+For the reference, here is the list of the said long-time Tox contributors
+running the infrastructure:
+
+*   [Impyy](mailto:impyy@tox.chat)
```
Comment (iphydf) [inline toktok/about.md:48]: What is this page about? It's not about the TokTok project, it seems.

File: toktok/faq.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+## How does Tox protect my privacy
+
+### Tox protects your privacy by
+
+-   Removing the need to rely on central authorities to provide messenger services
+-   Enforcing end-to-end encryption
+with [perfect forward secrecy](https://en.wikipedia.org/wiki/Forward_secrecy)
+as the default and only mode of operation for all messages
+-   Making your identity impossible to forge without the possesion of your
```
Comment (iphydf) [inline toktok/faq.md]: possession

File: toktok/faq.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+offline, as if you closed your client normally. They can't communicate
+with you any longer until you add them to your contacts list again.
+
+## Does Tox leak my IP address
+
+Tox makes no attempt to cloak your IP address when communicating with
+friends, as the whole point of a peer-to-peer network is to connect you
+directly to your friends. A workaround does exist in the form of tunneling
+your Tox connections through Tor. However, a non-friend user cannot easily
+discover your IP address using only a Tox ID; you reveal your IP address
```
Comment (iphydf) [inline toktok/faq.md:66]: This sentence seems to be wrong now.

@zugz.

File: toktok/faq.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+Check out our [public nodes list](https://nodes.tox.chat) for
+an updated list, including [machine-readable JSON output](https://nodes.tox.chat/json).
+
+## What codecs does Toxcore use for audio and video
+
+[Opus](http://opus-codec.org/) for audio,
+and [VP8](https://en.wikipedia.org/wiki/VP8) for video.
+Tox does not make use of SIP.
+
+## What is stopping people from tracking me through the public DHT
```
Comment (iphydf) [inline toktok/faq.md:109]: Is this section accurate? @zugz

**Review by iphydf — CHANGES_REQUESTED** [review]

File: toktok/about.md (hunk 1 above)
Comment (zoff99) [reply toktok/about.md:48]: its a close copy of the about page of tox.chat site.
it's supposed to be a start point, to move the site even closer together.
we can start from here, and then slowly correct anything thats no longer correct

File: toktok/faq.md (hunk 2 above)
Comment (zoff99) [reply toktok/faq.md]: done

File: toktok/faq.md (hunk 3 above)
Comment (zoff99) [reply toktok/faq.md:66]: its the corrected version (by zugz) from tox.chat. so should be ok?

**Comment by zugz** [comment]
* Friday, 2018-10-26 at 03:50 -0700 - iphydf <notifications@github.com>:

>+However, a non-friend user cannot easily
>+discover your IP address using only a Tox ID; you reveal your IP address
>
>This sentence seems to be wrong now.

With "easily", I think it's ok. I'd support including a link to #1152, 
though.

>+## What is stopping people from tracking me through the public DHT
>
>Is this section accurate? @zugz

Again, it's technically accurate, as it only claims to make things "more 
difficult" for an attacker. But again, linking to #1152 would make 
sense.

**Comment by zoff99** [comment]
@zugz what should be the text linking to https://github.com/TokTok/c-toxcore/issues/1152 ?
can you word something?

**Comment by zugz** [comment]
* Friday, 2018-10-26 at 17:36 +0000 - Zoff <notifications@github.com>:

>@zugz what should be the text linking to https://github.com/TokTok/c-toxcore/issues/1152 ?
>can you word something?

Probably best to leave this to a separate PR, once we're sure where the 
FAQ will live? I can draft something then.

**Pull request closed without merging**